### ISSUE TITLE
package.json errors

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/gravity-force-lab#20]
phetLibs: shows build-an-atom as a dependency, and it's missing axon and gravity-force-lab

preload: missing ../phetcommon/js/util/check-accessibility.js
